packages/generator/src/templateHelpers.ts

```diff
@@ -164,7 +164,7 @@ export const getReturnPayloadType = function(operation: any): string {
       dataTypes.push(
         res.payloads[0].schema.name.value() === "schema"
           ? "Object"
-          : res.payloads[0].schema.name.value()
+          : res.payloads[0].schema.name.value() + "T"
```

Comment:
why + "T" ? any doc-string on what this is?

Reply:
Without it we ran into a conflict with the className and some data types.  
For example if the type was "Customer" and the api was "Customer" they would exist in the same namespace and cause a conflict.  This happened 2 of the mercury APIs.  Doing this ensures Uniqueness.  Once exported within the class namespace we drop the T as it is no longer needed.

Reply:
I think it's a little cleaner to make something like this and call it
const convertToDataTypeName = (name: string) => `${name}T`;